pkg/xds/context/context.go

```diff
@@ -27,10 +26,15 @@ type ControlPlaneContext struct {
 }
 
 type MeshContext struct {
```

Comment:
There is a good comment about `MeshSnapshot` I think we need the same for `MeshContext`. But it feels hard for me to explain what's the actual difference between `MeshSnapshot` and `MeshContext`

Reply:
I think it's pretty easy, let me add a comment